concent_api/core/message_handlers.py

```diff
@@ -954,6 +957,69 @@ def store_message(
     return stored_message
 
 
+def handle_send_subtask_results_verify(
+    request,
+    subtask_results_verify: message.concents.SubtaskResultsVerify
+):
+    subtask_results_rejected = subtask_results_verify.subtask_results_rejected
+    report_computed_task = subtask_results_rejected.report_computed_task
+    compute_task_def = report_computed_task.task_to_compute.compute_task_def
+    validate_id_value(
+        compute_task_def['subtask_id'],
+        'subtask_id'
+    )
+    validate_golem_message_task_to_compute(report_computed_task.task_to_compute)
+    validate_golem_message_subtask_results_rejected(subtask_results_rejected)
+
+    client_public_key = decode_client_public_key(request)
+    other_party_public_key = decode_other_party_public_key(request)
+    current_time = get_current_utc_timestamp()
+
+    if Subtask.objects.filter(
+        subtask_id=compute_task_def['subtask_id'],
+        state__in=[
+            Subtask.SubtaskState.VERIFICATION_FILE_TRANSFER.name,  # pylint: disable=no-member
+            Subtask.SubtaskState.ADDITIONAL_VERIFICATION.name,     # pylint: disable=no-member
+        ]
+    ).exists():
+        return message.concents.ServiceRefused(
+            reason=message.concents.ServiceRefused.REASON.DuplicateRequest,
+        )
+    if current_time > subtask_results_rejected.timestamp + settings.ADDITIONAL_VERIFICATION_CALL_TIME:
```

Comment:
Please reverse the condition.

Reply:
do you mean:
`if not subtask_results_rejected.timestamp + settings.ADDITIONAL_VERIFICATION_CALL_TIME <= current_time` ?